scripts/launch/node.sh

```diff
@@ -112,6 +112,8 @@ educator_params="
 --student-api-no-auth 3BAyX5pNpoFrLJcP5bZ2kXihBfmBVLprSyP1RhcPPddm6Dw42jzEPXZz22
 --publication-period 15s
 --pdf-resource-path ../pdfs/template
+--cert-issuer-name "Grimpy Cat University"
+--cert-issuer-url example@gmail.com
```

Comment:
Don't you also need to update `full-config-sample.yaml`?

Reply:
Yes, it was written in the PR description as one of the things to change that I didn't have the time to do. 

It is done now, as soon as I'm done rebasing I'll push again. :smile:

Reply:
Oh, I was inattentive )

Reply:
No problem :smile: